### ISSUE TITLE
marble & marble-qt

### ISSUE THREAD

**Issue by anselmolsm** [clearlinux/distribution#1787]
Also another issue (according to me) regarding plasma desktop is kde desktop apps bundle containing 2 of some apps. For eg. there are two versions of the marble virtual globe app: one from kde and one from Qt. I am also not able to remove those apps via the swupd utility. Thanks for the response.

_Originally posted by @watson96 in https://github.com/clearlinux/distribution/issues/381#issuecomment-591357607_

**Comment by watson96** [comment]
I also want to uninstall packages like KMouth, Akonadi, KWrite, Kiten, Cantata etc. but it's not possible.

**Comment by fenrus75** [comment]
some of these are core parts of what defines KDE to be KDE....

**Comment by watson96** [comment]
ahem ahem... bloatware :wink:

**Comment by fenrus75** [comment]
yeah yeah
some are also mostly the library (akonadi) with a thin shim UI.... the
library is linked into the core of KDE stuff so you're not saving much

having said that I'm all for splitting the bundle more, just need to do it
smart so that we don't end up with a bad experience (not enough apps that
the KDE folks would consider essential to KDE)

**Comment by watson96** [comment]
> library is linked into the core of KDE stuff so you're not saving much

Yes that's something I noticed when searching the bundle files in KDE's desktop apps. KDE does seem to be somewhat heavier than GNOME even though those apps and libs weigh less. Can't blame those apps then.
